README.md

```diff
@@ -1,139 +1,5 @@
-
-### Cloning the test repository with Git
-
-After you have installed [Git][Git] you will need to clone the project to your hard drive. From your workspace directory run this command which will copy (clone) the project to your hard drive
-
-
-    git clone --recursive git://github.com/mozilla/Affiliates-Tests.git
-
-
-[Git]: http://git-scm.com/downloads
-
-### Installing Python packages
-
-You will need to install Selenium, py.test, unittestzero and some other project specific packages. Fortunately ```pip``` makes it easy to install all of these in one step. Let's start by installing pip:
-
-
-    sudo easy_install pip
-
-
-Now using pip we'll install the packages we need (which are listed in requirements.txt)
-
-    pip install -Ur requirements.txt
-
-If you are running on Ubuntu/Debian you will need to do following first:
-
-    sudo apt-get install python-setuptools
-
-to install the required Python libraries.
-
-
-### Running tests locally
-
-To run tests locally it's a simple case of calling ```py.test``` from the Affiliates-Tests directory.
-
-For more command-line options see the [pytest-mozwebqa documentation](https://github.com/mozilla/pytest-mozwebqa)
-
-### Virtual Environments (Optional/Intermediate level)
-
-The step above installs the packages globally on your operating system. Using ```virtualenv``` you can sandbox Python packages into virtual environments for each Mozilla project you work on. Follow our [Virtual Env guide][Virtual Environment] to get a virtual environment up and running. We recommend this practice.
-
-[Virtual Environment]: https://wiki.mozilla.org/QA/Execution/Web_Testing/Automation/Virtual_Environments
-
-
-Writing tests
--------------
-
-If you want to get involved and add more tests, then there's just a few things we'd like to ask you to do:
-
-1. Use the [template files][Template files] for all new tests and page objects
-2. Follow our simple [style guide][Style guide]
-3. Fork this project with your own GitHub account
-4. Add your test into the "tests" folder and the necessary methods for it into the appropriate file in "pages"
-5. Make sure all tests are passing and submit a pull request with your changes
-
-[Template files]: https://github.com/mozilla/mozwebqa-test-templates
-[Style guide]: https://wiki.mozilla.org/QA/Execution/Web_Testing/Docs/Automation/StyleGuide
-
-License
--------
-
-This software is licensed under the [MPL][MPL] 2.0:
-
-    This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
-    If a copy of the MPL was not distributed with this file, You can obtain one at
-    http://mozilla.org/MPL/2.0/.
-
-[MPL]: http://www.mozilla.org/MPL/2.0/
+This repository is no longer being actively maintained, and the content has
+been removed. Please use the repository history to access the previous content.
+We are not currently executing or developing tests for Affiliates in this repository.
+You can read more about MDN development and testing in the
```

Comment:
@justinpotts I think you can just get rid of the "You can read..." and "[project documentation]..." lines entirely, please!